Wip/getting started with almond (#63)
Stanford-Mobisocial-IoT-Lab/almond-cloud: doc/getting-started.md

## What is Almond?

Almond is the open virtual assistant that you can access through both of 

> your phone and the [web](/me). It can help you configure your devices, accounts, 

Reviewer: The user is unlikely to have an account at this point. How about:
```suggestion
your phone and the [web](/about/get-almond). It can help you configure your devices, accounts, 
```

Just like how Wikipedia stores knowledge about the world, 
Thingpedia stores knowledge about devices in the world. 
Wikipedia is organized around articles; 
Thingpedia is organized around devices, such as Twitter, a light bulb, or a thermostat.
It creates an interoperable web of devices and let users program them in 
natural language. 

## What can I do with Almond?

Almond draws the power from Thingpedia. 
It supports simple commands including 
actions (eg. `post on twitter`),
time-based rules (eg. `tweet every hour`),
one-time queries (eg. `search tweets`),
and standing queries (eg. `monitor new tweets`).
On top of one-time queries and standing queries, filters can be used to further refine 
the commands (eg. `search tweets from bob`, `monitor new tweets about almond`).

Almond also supports compound commands which interoperate functions from different devices.
For example, you can ask Almond to `send me a slack message if bob tweets`, 
`create a meme and tweet it`.

## Step-by-step example: Twitter to Almond

This example will guide you through filtering your Twitter feed and redirect
to Almond. At the end of the example, it will tell you about any tweet in your

> stream containing the hashtag "almondapp".

Reviewer: Having noticed that there are way to many almonds out there, maybe we should choose a different hashtag that is likely to pull up garbage? How about
```suggestion
stream containing the hashtag "#stanford".
```

"Device and Accounts". At this point, it probably does not know much, so we will 
teach him about Twitter.

To that extent, just click on
[Add New Account](/me/devices/create?class=online)
and then on
[Twitter Account](/me/devices/oauth2/com.twitter).

After you log in to Twitter and grant permission, you will be redirected to your
Almond page, which now includes Twitter.

### Step 3: Tell Almond what to do

Now Almond connects to your Twitter account. 
You can use the chat interface under [My Almond](/me) to write the following to Almond.

`notify me if i receive a tweet with hashtag #almondapp`

Follow the prompts to confirm, and congrats! You have your first standing query.
Now you will be notified whenever you open Web Almond of all tweets with hashtags \#almondapp.

Some other commands you can try:
- get recent tweets from users you follow: `get tweets`
- get your recent tweets: `get my tweets`
- post "hello" on your Twitter (if you don't mind): `tweet hello`
- follow our Twitter account: `follow @almondstanford on Twitter` 

Whenever you're tired of Almond telling you about Almond tweets, you can disable the

> rule by going to "Active Commands" section under [my Almond](/me), 

Reviewer: Capitalization:
```suggestion
rule by going to "Active Commands" section under [My Almond](/me), 
```
